includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -655,8 +655,13 @@ public function should_show_express_checkout_button() {
 
 		// If no SSL bail.
 		if ( ! $this->testmode && ! is_ssl() ) {
+			$server_details = [
+				'url'   => get_permalink(),
+				'https' => isset( $_SERVER['HTTPS'] ) ? wp_unslash( $_SERVER['HTTPS'] ) : '', // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+				'port'  => isset( $_SERVER['SERVER_PORT'] ) ? wp_unslash( $_SERVER['SERVER_PORT'] ) : '', // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+			];
 			// phpcs:ignore WordPress.PHP.DevelopmentFunctions
-			WC_Stripe_Logger::log( 'Stripe Express Checkout live mode requires SSL. ' . print_r( [ 'url' => get_permalink() ], true ) );
+			WC_Stripe_Logger::log( 'Stripe Express Checkout live mode requires SSL. ' . print_r( $server_details, true ) );
```

Comment:
We should use the new methods and add the details to the context:
```suggestion
			WC_Stripe_Logger::debug( 'Stripe Express Checkout live mode requires SSL', [ 'server_details' => $server_details ] );
```

Reply:
Good call, Diego. Sorry, I forgot about that 🤦